src/test/java/gov/usgs/owi/nldi/controllers/LinkedDataControllerTest.java

```diff
@@ -132,12 +146,32 @@ public void getCharacteristicDataTest() throws IOException {
 
 	@Test
 	public void getBasinTest() throws Exception {
+		when(lookupDao.getComid(anyString(), anyMap())).thenReturn(goodFeature());
+		doNothing().when(streamingDao).stream(anyString(), anyMap(), any());
```

Comment:
The streamingDao will fail because the 'goodFeature' doesn't actually exist.  Here I just force a mock streamingDao to not complain.  This test is now the happy path - the previous test of null input values doesn't really test a situation we would see (and its tested directly by getComId).

Reply:
As noted above, this class is pure unit tests of the controller itself. Full fledged integration tests against the database are in LinkedDataControllerOtherIT. A not found test there may be appropriate.

Reply:
Good thought - I'll add that on Friday.